example/setup.py

```diff
@@ -18,7 +18,7 @@
     name="my-example",
     version="0.1.0",
     packages=find_packages(include=["my_example", "my_example.*"]),
-    python_requires=">=3.7, <3.9",
+    python_requires=">=3.7, <3.13",
```

Comment:
tested?

Reply:
Note this is an exclusive `<3.13`. This was tested up to 3.12 including as declared.

Reply:
Python 3.13 support is in https://github.com/awslabs/aws-codeseeder/pull/294.

Reply:
Nice